apps/i18n/common/en_us.json

```diff
@@ -586,6 +586,9 @@
   "createAccountToShareDescription": "You must create a Code.org account before you can share and publish your project. Creating an account will also let you save your progress and continue to work on your project later.",
   "createBlocklyBehavior": "Create a Behavior",
   "createClassSections": "Create class sections",
+  "createClassSectionsToAssign": "Create class section to assign curriculum",
```

Comment:
Nit: "assign a curriculum" or "assign curricula" ?

Reply:
Good call! Changed it to `assign a curriculum` because they're only assigning one at a time